command/core.py

```diff
@@ -13,14 +13,15 @@ def __init__(self):
     def handle_app_mention(self, event_data):
         """Handle the events associated with mentions of @rocket."""
         message = event_data["event"]["text"]
+        user = event_data["event"]["user"]
```

Comment:
Once we have this running we'll want to double check what format `event_data["event"]["user"]` is (e.g. is it the actual `@rwblickhan`, or something like `<@U12345>`?).